templates/competitorResults.html

```diff
@@ -1,17 +1,31 @@
 <template name="participantResults">
   <div class="container">
-
-    <ul>
-      {{#each results}}
-        <li>
-          {{! TODO - perhaps combine this and the code in roundResults into a template? }}
-          {{#each solves}}
-            {{clockFormat this}}
-          {{/each}}
-          best: {{clockFormat best}}
-        </li> 
-      {{/each}}
-    </ul>
+    <h1>{{competitorName}}</h1>
+    <h3><a href={{wcaProfileUrl}}>WCA Profile</a></h3>
+    <table class="table table-striped table-bordered table-hover">
```

Comment:
I copy and pasted it from somewhere, not really sure what it does, so I just removed it. table-bordered was what I wanted